### PR TITLE
Add Windows.Analysis.EvidenceOfDownload

### PR DESCRIPTION
**Pull request by msoheem**
Simple artifact to detect downloaded files in a directory provided as paramerter to the artifact.
This artifact uses Zone.Identifier alternate stream created with files downloaded on windows machines from different security zones.

### CODE REVIEW

File: artifacts/definitions/Windows/Analysis/EvidenceOfDownload.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - https://www.sans.org/security-resources/posters/windows-forensic-analysis/170/download
+   
+author: M.Soheem @msoheem
+
+# Can be CLIENT, CLIENT_EVENT, SERVER, SERVER_EVENT
+type: CLIENT
+
+parameters:
+   - name: DirectoryPath
+     default: "/C:/Users/dfir/Downloads/"
```
Comment (scudette) [inline artifacts/definitions/Windows/Analysis/EvidenceOfDownload.yaml:19]: Let's default it to all users? C:/Users/*/Downloads/

File: artifacts/definitions/Windows/Analysis/EvidenceOfDownload.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+sources:
+  - precondition:
+      SELECT OS From info() where OS = 'windows'
+
+    query: |
+     SELECT get(item=split(string=FullPath, sep=":Zone.Identifier"),member='0') as DownloadedFilePath,
+            hash(path=get(item=split(string=FullPath, sep=":Zone.Identifier"),member='0')) as FileHash,
```
Comment (scudette) [inline artifacts/definitions/Windows/Analysis/EvidenceOfDownload.yaml:27]: You can refactor it into two steps then you dont need to split twice:
```sql
LET X = SELECT split(string=FullPath, sep=":")[0] AS DownloadedFilePath, ... 
   FROM glob(...)

SELECT DownloadedFilePath, hash(path=DownloadedFilePath) AS FileHash
FROM X
```

File: artifacts/definitions/Windows/Analysis/EvidenceOfDownload.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            read_file(filename=FullPath,accessor="ntfs") as ZoneIdentifierContent
+     FROM glob(globs=path_join(components=[DirectoryPath,"**/*:Zone.Identifier"]),accessor="ntfs")
```
Comment (scudette) [inline artifacts/definitions/Windows/Analysis/EvidenceOfDownload.yaml:29]: We might want to also add the URL to this query (from https://docs.google.com/presentation/d/1SUPH_D1GPQ7knIfCbj9wGSPuLTSltXjPV95teCY47LU/edit#slide=id.gc2be8b3c533c783_0 
)

File: artifacts/definitions/Windows/Analysis/EvidenceOfDownload.yaml (hunk 2 above)
Comment (msoheem) [reply artifacts/definitions/Windows/Analysis/EvidenceOfDownload.yaml:27]: Sorry first artifact to write :)
Great tips Mike

File: artifacts/definitions/Windows/Analysis/EvidenceOfDownload.yaml (hunk 3 above)
Comment (msoheem) [reply artifacts/definitions/Windows/Analysis/EvidenceOfDownload.yaml]: Okay I placed the content of the alternate stream as not all downloading apps add the URL for example FireFox does not add the URL. however, I'll try to parse it

**Pull request merged**